cmd/app.go

```diff
@@ -18,28 +18,26 @@ package cmd
 import (
 	"github.com/WSBenson/goku/internal/app"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
+var port string
```

Comment:
you don't need this. on line 32, you can just instantiate when you set `port`'s value with `:=`